dataactcore/migrations/versions/e3477fa42630_submission_window_type_lookup.py

```diff
@@ -0,0 +1,50 @@
+"""Submission window type lookup
```

Comment:
I just realized this isn't really an accurate message. A more accurate message would be "Adding application_type table, updating submission_window table"